src/Microsoft.AspNetCore.Proxy/ProxyMiddleware.cs

```diff
@@ -1,22 +1,28 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using Microsoft.AspNetCore.Builder;
```

Comment:
The default settings are bad... Not your fault. You can change them here:

![image](https://cloud.githubusercontent.com/assets/95136/23283088/0c232306-f9d9-11e6-8ea7-47a25f748329.png)